src/ratings-and-reviews/components/ReviewBreakdown.jsx

```diff
@@ -1,39 +1,56 @@
-import React from 'react';
+import React, {useState, useEffect} from 'react';
 import {reviewsTotal} from '../index.jsx';
 
-const ReviewBreakdown = ({breakdown}) => (
-  <div className="review-breakdown">
-    <div>
-      <label className="review-breakdown-label">
+const checkStarFilters = (num, starFilters, setStarFilters) => {
+  if (starFilters.indexOf(num) === -1) {
+    setStarFilters(starFilters.concat([num]));
+  } else {
+    setStarFilters(starFilters.filter(number => number !== num));
+  }
+};
```

Comment:
I wonder if this might work better as an array of five booleans. Something to play around with next week, maybe